lib/load.php

```diff
@@ -28,7 +28,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 	return ! empty( $experiments[ $name ] );
 }
 
-// These files only need to be loaded if within a rest server instance
+// These files only need to be loaded if within a rest server instance.
 // which this class will exist if that is the case.
 if ( class_exists( 'WP_REST_Controller' ) ) {
 	if ( ! class_exists( 'WP_REST_Block_Editor_Settings_Controller' ) ) {
```

Comment:
@matiasbenedetto what flag is there to know that the Fonts Library frontend work is available and those the backend should be loaded into memory? Also, should the Fonts Library feature be something that the user / system turns on, at least until it's fully ready to replace the Fonts API and current font workflow?

Context: 
- The Fonts Library backend (this PR) is not needed in memory until the frontend is available.
- The server-side `@font-face` generator and printer logic has 2 different implementations (Font Face and Fonts API) but only 1 of those should be loaded into memory: 
    - Font Face needs the Fonts Library.
    - Fonts API is not compatible with the Fonts Library.
 - Getting this PR merged before the frontend work is beneficial. How so?
    - Allows for smaller improvements and iterative work, i.e. in both the backend and frontend.
    - Adds the backend functionality for the frontend work, to provide full testing environment for the feature.
    - Reduces scope creep, which can happen the longer a PR stays open.

Reply:
Spoke with @matiasbenedetto. What's needed is a way for local dev / testing and automated tests to enable the Fonts Library. We're proposing to use a constant `FONTS_LIBRARY_ENABLE` that is defined in the `wp-config.php` or test file.

Reply:
Added here: https://github.com/WordPress/gutenberg/pull/52704/commits/60722143c24003a9b1d7619cf4ea76284efd86ca

Reply:
Commit https://github.com/WordPress/gutenberg/pull/52704/commits/7586418e418233a157565fa84371c174837f1f0d turns on the tests for it ✅